### DISCUSSION TITLE
Barycentric coordinates of the points generated by the SamplePoints method

### DISCUSSION THREAD

**Discussion by dhorka** [pyg-team/pytorch_geometric#4263] [Q&A]
Hi,
I saw that pytorch geometric has the transformation called SamplePoints that samples points from a mesh. I was wondering if there is an efficient way in pytorch geometric to recover the barycentric coordinates of each of the sampled points or at least to know to which face belongs each point. 

Thanks!

**Comment by rusty1s** [comment]
Currently no. We would need to add support for it directly within `SamplePoints` (perhaps with some flags to indicate which additional data to create). Let me know if you have interest in contributing this feature.
